shared/js/remoteCursorStore.js

```diff
@@ -44,7 +45,8 @@ loop.store.RemoteCursorStore = (function() {
     getInitialStoreState: function() {
       return {
         realVideoSize: null,
-        remoteCursorPosition: null
+        remoteCursorPosition: null,
+        shouldRenderCursor: true
```

Comment:
At first I was going to suggest just replacing remoteCursorPosition when hasVideo goes false, but keeping the position separate would allow it to persist its location to show. Although it's not necessarily useful to have the cursor reshow where it was before the sharing was paused, but that's a minor issue.